steamclog/src/main/java/com/steamclock/steamclog/Steamclog.kt

```diff
@@ -180,19 +177,15 @@ object SteamcLog {
      * Log files may be spread out across multiple files; this method will join
      * all log files into a single String.
      */
-    suspend fun getFullLogContents(): String? {
+    fun getFullLogContents(): String? {
         return externalLogFileTree.getLogFileContents()
     }
 
     /**
-     * Returns the latest log file, based on the autoRotateConfig setting.
+     * Returns a list of ALL log files;
      */
-    fun getLastLogFile(): File? {
-        return externalLogFileTree.getExternalFile()
-    }
-
-    fun deleteLogFile() {
```

Comment:
We don't actually support manual delete due to auto rotation; removing to avoid confusion